scripts/linters/js_ts_linter.py

```diff
@@ -1139,6 +1139,109 @@ def _check_comments(self):
             summary_messages.append(summary_message)
         return summary_messages
 
+    def _check_angular_services_index(self):
+        """Finds all @Injectable classes and makes sure that they are added to
+            Oppia root and Angular Services Index.
+
+        Returns:
+            all_messages: str. All the messages returned by the lint checks.
+        """
+        def get_injectable_class_name(file_content):
+            """Extarcts the class name from a file that has an Injectable
+                class.
+
+            Args:
+                file_content: str. File content of the file that has an
+                    Injectable class.
+
+            Returns:
+                tuple(str, str). A two-tuple of class name and class name in
```

Comment:
Is this true? I only see a `return class_name`*

Reply:
Changed!